$NetBSD: patch-aa,v 1.3 2025/03/30 15:39:58 wiz Exp $

Don't screw with libc feature hiding macros in ways that break the
build.

--- src/definitions.h.orig	2000-05-25 16:00:15.000000000 +0000
+++ src/definitions.h
@@ -31,6 +31,9 @@
 # define __USE_BSD
 #endif
 
+#if !defined(__NetBSD__) && !defined(__FreeBSD__) && !defined(__DragonFly__) \
+ && !defined(__OpenBSD__)
+
 #ifndef __EXTENSIONS__
 # define __EXTENSIONS__
 #endif
@@ -62,6 +65,7 @@
 # endif
 #endif
 
+#endif /* __NetBSD__ */
 
 
 
